### DISCUSSION TITLE
Help getting filters to work

### DISCUSSION THREAD

**Discussion by rasmusthog** [FreshRSS/FreshRSS#5621] [Q&A] [answered]
I am having a hard time getting the filters using search fields to properly work, even what I would assume is the most basic usage. I am assuming it is just something I have overlooked or misunderstood, and that I can easily be pointed in the right direction here:

There is a local newspaper I follow whose feed is very active, and a lot of it is irrelevant to me, so I want to filter out a lot of the categories I don't care about, such as sports. Luckily, they sort their categories in the URL, so for example sports news would have URLs looking like this: www.example.com/sport/code/title-of-the-article

I have simply tried adding 'inurl:sport' in the 'Mark article as read...' box under in the feed management panel. Even so, all articles redirecting to www.example.com/sport/etc arrives as unread.

**Comment by Alkarex** [comment]
Hello,
What you have done with `inurl:sport` looks correct. I will check whether I can reproduce a bug.

**Reply by Frenzie** [reply]
What exactly do you mean by "redirecting to `example.com/sport`"? That phrasing raises some suspicions. :laughing:

**Reply by Alkarex** [reply, marked as answer]
I can confirm that `inurl:example` works as expected. As Frenzie suspects, it could be that the URL in the feed is different than the final URL after clicking on the article.

**Reply by rasmusthog** [reply]
Thanks for checking, and thanks for confirming that I've done it as it should be done. I'll try to more systematically assess this throughout the day, and adding new filters to other feeds as well.

The phrasing "redirecting to" is simply an inaccuracy on my part. I mean that the URL for the article is exactly example.com/sport/code/title-of-the-article. There appears to be no difference in the URL in the FreshRSS feed and where I am sent, and the URL in the feed does contain "sport".

**Reply by Alkarex** [reply]
If you can share the URL of the feed, we might be able to debug some more. Otherwise no worries

**Reply by rasmusthog** [reply]
Ok, it seems to work for me now. I think I might've mistakenly thought that a batch of articles was fetched after I applied the filter, when they instead probably had already been fetched. I just received four sports articles that all were marked as read on arrival. Now I'll dig a bit deeper into the filters to see if I can game the feeds that are not as willing to share the categories of their articles in the URL :) 

Sorry to bother you, and thanks for making a great service!
